daemon/bindata.go

```diff
@@ -1,5 +1,9 @@
 // Code generated by go-bindata.
 // sources:
+// ../bpf/.bpf_lxc.c.swn
```

Comment:
Can you add `*.swn` and `*.swp` into `.gitignore` and regenerate the `bindata.go`